InscryptionCommunityPatch/Card/LatchFix.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using DiskCardGame;
+using HarmonyLib;
+using System.Reflection;
+using UnityEngine;
+
+namespace InscryptionCommunityPatch.Card
+{
+    [HarmonyPatch]
+    public static class LatchFix
```

Comment:
Should be merged into the existing latch fix class so all the code to make latch work is in one place.